pkg/security/security.go

```diff
@@ -131,6 +131,12 @@ var (
 	TokenAudiences = strings.Split(env.Register("TOKEN_AUDIENCES", "istio-ca",
 		"A list of comma separated audiences to check in the JWT token before issuing a certificate. "+
 			"The token is accepted if it matches with one of the audiences").Get(), ",")
+
+	// DefaultKubernetesAudience specifies the default audience used by the kubernetes token issuer.
```

Comment:
https://github.com/maistra/istio/pull/418#issuecomment-899677060
:(

Reply:
It seems that we no longer need this commit, because it's already implemented in the upstream Istio. Look at this PR: https://github.com/istio/istio/pull/34294.

Reply:
We have a bunch identified so maybe it's time to do it all at once?

Reply:
What do you mean? This feature is already implemented in upstream.

Reply:
Awesome! One less commit to carry over :)

Reply:
> What do you mean? This feature is already implemented in upstream.

@jewertow I misinterpreted `:(` comment from @jwendell

Reply:
That's gone now.